tox.ini

```diff
@@ -21,6 +19,7 @@ commands = coverage report
 
 [testenv:typing]
 description = Run mypy.
+allowlist_externals = mypy
```

Comment:
You can delete this now that mypy should be installed in the environments thanks to the change in `[testenv]`.